Compiler/src/abstractinterpretation.jl

```diff
@@ -2637,21 +2637,14 @@ function abstract_call_known(interp::AbstractInterpreter, @nospecialize(f),
         elseif f === Core.getfield && argtypes_are_actually_getglobal(argtypes)
             return Future(abstract_eval_getglobal(interp, sv, si.saw_latestworld, argtypes))
         elseif f === Core.isdefined && argtypes_are_actually_getglobal(argtypes)
```

Comment:
```suggestion
        elseif f === Core.isdefined && argtypes_are_actually_getglobal(argtypes) && 3 <= length(argtypes) <= 4
```

Reply:
`argtypes_are_actually_getglobal` has the length check.